src/nvim/popupmnu.c

```diff
@@ -219,7 +220,9 @@ void pum_display(pumitem_T *array, int size, int selected)
     if (curwin->w_p_rl) {
       pum_width = pum_col - pum_scrollbar + 1;
     } else {
-      pum_width = Columns - pum_col - pum_scrollbar;
+      assert(Columns - pum_col - pum_scrollbar >= INT_MIN
```

Comment:
An even shorter version, like `IN_BOUNDS(INT, <value>)` is planned, but not done yet.
I prefer to tackle that as a separate task, as there are a lot of previously introduced asserts that should be refactored that way.
We also had in mind macros to check for arithmetic overflow, like `IN_BOUNDS_SUM(INT, <value_1>, <value_2>)`. Again, that will be a separate task.